assistant/assistant.py

```diff
@@ -260,11 +260,11 @@ async def on_message(body: str) -> None:
     logger.info(f"[ASSISTANT] -> message received: {body}")
     from_client = json.loads(body)
 
-    # Handle igonore media message
+    # Handle ignore media message
     media = from_client.get("media")
     if media:
         logger.info("[ASSISTANT] -> skipping whisper because of media message")
-        return True
+        return False
```

Comment:
Should this not be `return`? The function specifies `None` as the return type, not `boolean`.